README.md

```diff
@@ -181,7 +181,7 @@ own stylesheet with the specific overrides.
 
 ## Internationalization
 
-Torch comes with `.po` files for `en`, `ru` and `es` locales. If you are using
+Torch comes with `.po` files for `en`, `ru` `es` and `de` locales. If you are using
```

Comment:
Please add a missing comma to this sentence

> Torch comes with `.po` files for `en`, `ru`, `es`, and `de` locales.

Reply:
```suggestion
Torch comes with `.po` files for `en`, `ru`, `es` and `de` locales. If you are using
```

Reply:
@cpjolicoeur Ops, done